glibc has changed the error code it returns from '1' to '2'.  The 
included grep regex library returns '1' still though, so spencer 
test #55 incorrectly fails.  For now, we update the test to match 
the glibc regex since that is what we're using ;).

http://bugs.gentoo.org/show_bug.cgi?id=76192
http://linuxfromscratch.org/pipermail/lfs-dev/2003-February/032543.html

--- tests/spencer1.tests.orig	2005-01-06 00:17:34.550645568 -0500
+++ tests/spencer1.tests	2005-01-06 00:18:43.682135984 -0500
@@ -55 +55 @@
-1@(*)b@-
+2@(*)b@-
